test/cassowary_test.dart

```diff
@@ -38,53 +38,40 @@ void main() {
     final v2 = Param(10);
     final v3 = Param(22);
 
-    expect(v1 is Param, true);
```

Comment:
I'm curious as to why these lines are being deleted

Reply:
I did so, because of broken analyze pipeline

https://github.com/google/cassowary.dart/runs/5563456285?check_suite_focus=true
````
Run dart analyze --fatal-infos
Analyzing cassowary.dart...

warning - pubspec.yaml:4:1 - The 'author' field is no longer used and can be removed. Try removing the field. - deprecated_field
   info - test/cassowary_test.dart:[7](https://github.com/google/cassowary.dart/runs/5563456285?check_suite_focus=true#step:6:7):1 - Sort directive sections alphabetically. - directives_ordering
   info - test/cassowary_test.dart:[8](https://github.com/google/cassowary.dart/runs/5563456285?check_suite_focus=true#step:6:8):8 - The import of 'package:matcher/matcher.dart' is unnecessary because all of the used elements are also provided by the import of 'package:test/test.dart'. Try removing the import directive. - unnecessary_import
   info - test/cassowary_test.dart:41:[12](https://github.com/google/cassowary.dart/runs/5563456285?check_suite_focus=true#step:6:12) - Unnecessary type check; the result is always 'true'. Try correcting the type check, or removing the type check. - unnecessary_type_check
   info - test/cassowary_test.dart:42:12 - Unnecessary type check; the result is always 'true'. Try correcting the type check, or removing the type check. - unnecessary_type_check
   info - test/cassowary_test.dart:43:12 - Unnecessary type check; the result is always 'true'. Try correcting the type check, or removing the type check. - unnecessary_type_check
   info - test/cassowary_test.dart:48:12 - Unnecessary type check; the result is always 'true'. Try correcting the type check, or removing the type check. - unnecessary_type_check
   info - test/cassowary_test.dart:55:12 - Unnecessary type check; the result is always 'true'. Try correcting the type check, or removing the type check. - unnecessary_type_check
   info - test/cassowary_test.dart:58:12 - Unnecessary type check; the result is always 'true'. Try correcting the type check, or removing the type check. - unnecessary_type_check
   info - test/cassowary_test.dart:60:12 - Unnecessary type check; the result is always 'true'. Try correcting the type check, or removing the type check. - unnecessary_type_check
   info - test/cassowary_test.dart:67:12 - Unnecessary type check; the result is always 'true'. Try correcting the type check, or removing the type check. - unnecessary_type_check

... and so on
````
Also not sure about removing 'author' field in pubspec. 

Say please to me how to do right and I will fix.